.meta/transforms/dedupe.toml

```diff
@@ -0,0 +1,41 @@
+[transforms.dedupe]
+title = "Dedupe events"
+allow_you_to_description = "prevent duplicate Events from being outputted by using an LRU cache"
+beta = false
+common = false
+function_category = "filter"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "transform", name: "dedupe") %>
+
+[transforms.dedupe.options.fields]
+type = "table"
+description = "Options controlling what fields to match against"
+required = true
+
+[transforms.dedupe.options.fields.children.match]
+type = "[string]"
+common = true
+examples = [["field1", "field2"]]
+default = ["timestamp"]
+description = "The field names considered when deciding if an Event is a duplicate. Incompatible with the \"fields.ignore\" option."
+
+[transforms.dedupe.options.fields.children.ignore]
+type = "[string]"
+common = true
+examples = [["field1", "field2"]]
+description = "The field names to ignore when deciding if an Event is a duplicate. Incompatible with the \"fields.match\" option."
+
+
+[transforms.dedupe.options.cache]
+type = "table"
+description = "Options controlling how we cache recent Events for future duplicate checking."
+
+[transforms.dedupe.options.cache.children.num_entries]
```

Comment:
Can we rename this this `num_events` for consistency with the `buffer.num_events` naming?

Reply:
done